.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@ default_stages: [commit]
 
 repos:
   - repo: https://github.com/charliermarsh/ruff-pre-commit
-    rev: v0.0.272
+    rev: v0.1.6
```

Comment:
```suggestion
    rev: v0.1.5
```

Reply:
I couldn't see from the linked thread which version ruff fixes this in. Feel free to go for 0.1.6 if aligned with dev-requirements.txt